infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -94,10 +96,10 @@
     private Future failureDetectorFuture = null;
     private AtomicBoolean recovered = new AtomicBoolean(false);
 
-    @Override
-    public boolean isServerReady() {
-        return true;
-    }
+    @Getter
+    private volatile boolean sequencerBootstrapped = false;
+    private static final int sequencerBootstrapRetries = 3;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)